packages/@aws-cdk/aws-apprunner/test/service.test.ts

```diff
@@ -391,6 +391,76 @@ test('custom environment secrets and start commands are allowed for imageConfigu
   });
 });
 
+
+test('custom environment variables and secrets can be added without first defining them in props', () => {
```

Comment:
Can we split this into two tests, one for each add*() method? If someone breaks one of these methods but not another, having the name of the test match the method that was broken will make it easier to debug.

Reply:
Done